app/queries/assigned_patients_query.rb

```diff
@@ -1,10 +1,12 @@
 class AssignedPatientsQuery
-  # Returns ALL assigned counts for a region
+  # Returns hypertensive assigned counts for a region
```

Comment:
@rsanheim This comment looks superfluous, can I get rid of it?

Reply:
Yup 👍